cooperator/tests/cooperator_test_mixin.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from datetime import date, datetime, timedelta
+
+
+class CooperatorTestMixin:
+    @classmethod
+    def set_up_cooperator_test_data(cls):
+        # accounting data needs to be created even if created in module data
+        # because when launching tests, accounting data will be deleted when odoo loads
+        # a test chart of account.
+        # cf load_for_current_company in chart_template.py in account module
+
+        # if these properties are not set, odoo thinks the chart of account was not
+        # loaded. The test CAO does not set those properties
+        # cf AccountingTestCase
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+
+        cls._ensure_account_property("res.partner", "property_account_receivable_id")
+        cls._ensure_account_property("res.partner", "property_account_payable_id")
+        cls._ensure_account_property("product.template", "property_account_income_id")
+
+        account_model = cls.env["account.account"]
+        cls.company = cls.env.user.company_id
+        cls.company.coop_email_contact = "coop_email@contact.net"
```

Comment:
i think that it would be better to use [reserved domain names](https://www.iana.org/domains/reserved) (like `example.org`) for dummy data instead of real domains. i’m not sure which one is the more appropriate, though, because the rfc does mention documentation and examples, but not test data.